extension/jaegerremotesampling/fixtures/TestServiceNoPerOperationStrategiesDeprecatedBehavior_ServiceA.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
where are any of these used?

Reply:
these are currently used in the provider_test file taken from jaeger repo

Reply:
+1